environmentmodel/environment-model_shacl.ttl

```diff
@@ -0,0 +1,288 @@
+@prefix environmentmodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environmentmodel/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+environmentmodel:environmentModelShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environmentmodel:QuantityShape ;
+            sh:order 2 ;
+            sh:path environmentmodel:Quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environmentmodel:QualityShape ;
+            sh:order 3 ;
+            sh:path environmentmodel:Quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environmentmodel:ProjectShape ;
+            sh:order 0 ;
+            sh:path environmentmodel:Project ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environmentmodel:FormatShape ;
+            sh:order 1 ;
+            sh:path environmentmodel:Format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environmentmodel:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path environmentmodel:Georeference ] ;
+    sh:targetClass environmentmodel:environmentModel .
```

Comment:
PascalCase class
-> `EnvironmentModel`

Please correct this for the others also